src/style.css

```diff
@@ -0,0 +1,92 @@
+/*Add custom font*/
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir.ttc);
+}
+
+/* styles shered with the both search bar */
+.search-bar {
+  display: flex;
+  margin-top: 20px;
+  background: rgb(255, 255, 255);
+  border: 1px solid rgb(226, 232, 238);
+  box-shadow: 0 2px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+/* Search icon, works like a button */
+.search-icon-big {
+  border: none;
+  padding: 24px 18px 25px 25px;
+}
+
+.input-big-bar {
+  padding: 0;
+  flex: 1;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-weight: 300;
+  color: rgb(61, 78, 97);
+  border: none;
+}
+
+.input-big-bar:focus {
+  outline: none;
+  background: linear-gradient(180deg,
+  rgb(255, 255, 255) 0%, rgb(246, 246, 247) 100%)
+   ;
+  font-weight: 600;
+  text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
```

Comment:
you have duplicated styles on line 80-87. it would be better to create a common class as `.input-bar` and add common styles fo that selector. then just use `.input-small-bar` to modify styles which are different for smaller input